### ISSUE TITLE
Restore HTTP access to liquibase schemas

### ISSUE THREAD

**Issue by zambrovski** [liquibase/liquibase#2449]
Hi folks,

the error re-appeared again. Today, the call `curl -v http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd` delivers 301 moved permanently (Location: https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd).

SAX Parsers in Keycloak for example are not able to get them. Could you please re-enable the unsafe http access?

Thanks,

Simon

_Originally posted by @zambrovski in https://github.com/liquibase/liquibase/issues/1153#issuecomment-1026597782_

**Comment by leitus** [comment]
Hi,
we have the same problem since yesterday
The error is 
init method failed; nested exception is liquibase.exception.ChangeLogParseException: liquibase.exception.SetupException: Error parsing line 2 column 35 of classpath:config/liquibase/changelog/20190711143117_added_entity_Paycheck.xml: s4s-elt-character: Non-whitespace characters are not allowed in schema elements other than 'xs:appinfo' and 'xs:documentation'. Saw '301 Moved Permanently'.
When call http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd

Thanks

Leo

**Comment by AleksanderGrzybowski** [comment]
Please bring it back. I already commented in the other issue, but I can't easily update the URL without redeploying a lot of stuff, I'd like to avoid it. Or tell me that it won't be fixed, so I can prepare for it. :) Thanks!

**Comment by molivasdat** [comment]
Hi @zambrovski, @leitus and @AleksanderGrzybowski .
Thanks for the update.
We are looking into the 301s currently.
What versions of Liquibase are each of you using?
Later versions of Liquibase look locally for the xsds.

**Comment by AleksanderGrzybowski** [comment]
Yes, that may be a pretty old version, it is `version 3.8.9 #73 built at Mon Apr 06 09:32:58 UTC 2020`.

**Comment by lbanas** [comment]
try this : https://github.com/liquibase/liquibase/issues/2448#issuecomment-1026873548

**Comment by molivasdat** [comment]
Hi @zambrovski, @leitus and @AleksanderGrzybowski @lbanas This should be fixed now.

**Comment by AleksanderGrzybowski** [comment]
Thanks for your amazing support! Kudos! :)

**Comment by leitus** [comment]
Thank you!
